src/app/service/squad/LaunchSquad.ts

```diff
@@ -65,15 +65,19 @@ const getTitle = async (member: GuildMember, ctx?: CommandContext): Promise<void
 
 	collector.on('end', async (_, reason) => {
 
-		// if getTitlePrompt is not the last message, time out silently.
-		if ((getTitlePrompt.id === dmChannel.lastMessage.id) && (reason === 'time')) {
-			Log.debug('squadUp getTitle message collector timed out');
+		try {
+			// if getTitlePrompt is not the last message, time out silently.
+			if ((getTitlePrompt.id === dmChannel.lastMessageId) && (reason === 'time')) {
+				Log.debug('squadUp getTitle message collector timed out');
 
-			try {
-				await dmChannel.send('The conversation timed out.');
-			} catch (e) {
-				Log.debug(`Squad getTitle collector timed out, unable to send dm, error msg: ${e}`);
+				try {
```

Comment:
Is there a way to approach this without the numerous nested `try ... catch` statements? It appears the sole reason for the nested `try ... catch` is to capture logging on a failed `await`. With a little work, I think this file can be refactored to manage the logging without all the nests.

Reply:
I actually did that to prevent it from crashing by failed DMs. Just checked and realized that was fixed globally now, so removed a lot of try catch. Thanks for mentioning it.